### PR TITLE
[Dominion] [Site] Footer

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

Fixes #162380525

## Type of change
- [X] New feature (non-breaking change which adds functionality)

## Screenshots

image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: dominion/frontend/src/assets/css/index.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+html {
```
Comment (kilemensi) [inline dominion/frontend/src/assets/css/index.css]: We should try not to use general css files like this. All components should have styles defined in the components themselves.

File: dominion/frontend/src/component/Footer/Partners.js [hunk 2]
```diff
@@ -1,15 +1,64 @@
...
+        </Grid>
+        <Grid item xs={2}>
+          <img src={twaweza} alt="Twaweza" className={classes.img} />
+        </Grid>
+        <Grid item xs={2}>
+          <img src={bmf} alt="Bill and Melinda Gates" className={classes.img} />
+        </Grid>
+        <Grid item xs={2}>
+          <img src={icfj} alt="Icfj" className={classes.img} />
+        </Grid>
```
Comment (kilemensi) [inline dominion/frontend/src/component/Footer/Partners.js]: This design works well for large screens but for small/extra small screens, the partners logos become to small and disappear completely.

screenshot from 2019-01-29 08-11-55

File: dominion/frontend/src/component/Footer/Partners.js [hunk 3]
```diff
@@ -1,15 +1,64 @@
 import React from 'react';
-import { Grid, Typography } from '@material-ui/core';
+import PropTypes from 'prop-types';
```
Comment (kilemensi) [inline dominion/frontend/src/component/Footer/Partners.js:2]: Any reason Partners is part of Footer? Isn't it part of the page and hence should be its own component i.e. `src/components/Partners.js`

If it **is** part of Footer, then Footer should include Partners and pages should only include Footer i.e. Partners shouldn't be used outside the Footer.

File: dominion/frontend/src/component/Footer/SocialMedia.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
```
Comment (kilemensi) [inline dominion/frontend/src/component/Footer/SocialMedia.js]: Same as Partners. Either

 * Part of Footer: Then Footer should included it and **no other component/page** should know/use it, or
 * Component of its own: Move it to components folder ie. `src/components/SocialMedia.js`.

File: dominion/frontend/src/component/Footer/index.js [hunk 5]
```diff
@@ -1,18 +1,163 @@
...
+    padding: '2rem',
+    backgroundImage: `url(${background})`,
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: 'cover',
+    [theme.breakpoints.up('md')]: {
+      height: '50vh'
+    }
+  },
+  title: { color: '#f1f1ed', fontWeight: 'bold', opacity: '0.6' },
+  body: { color: '#f1f1ed', opacity: '0.6', paddingTop: '3rem' },
```
Comment (kilemensi) [inline dominion/frontend/src/component/Footer/index.js]: I think lets start from the get go using **_theme_** for these common styles.

**Review by kilemensi — CHANGES_REQUESTED** [review]

File: dominion/frontend/src/component/Footer/index.js (hunk 5 above)
Comment (nerdlet) [reply dominion/frontend/src/component/Footer/index.js]: @kilemensi Created a PR with the color palette and typography

**Review by kilemensi — CHANGES_REQUESTED** [review]
There is still issues on Small/Extra small screens:

They either look too big:

screen shot 2019-01-30 at 14 31 20

Or too small:

screen shot 2019-01-30 at 14 31 32

PS
---------------------------------------------------------

To include a component `components/Footer/index.js`, use:
```javascript
import Footer from '../component/Footer';
```
and not:

```javascript
import Footer from '../component/Footer/Index';`
```

**Comment by nerdlet** [comment]
Hi @kilemensi Improved the images. This is how they look like presently

image

**Comment by kilemensi** [comment]
> Hi @kilemensi Improved the images. This is how they look like presently
> 
> image

Still look too big (see the iPad image)... But we'll sort it out when the page is complete.

**Pull request merged**